### ISSUE TITLE
Some GraphQL requests are broken.

### ISSUE THREAD

**Issue by krzesu0** [carpedm20/fbchat#441]
## Description of the problem
`graphql_request` *sometimes* throws an Exception.

## Code to reproduce
Eg.
```py
from fbchat import Client
from getpass import getpass
client = Client("whatever", getpass())
# This will throw
x = client.searchForThreads("whatever")
# This wont
x = client.fetchThreadInfo("whatever")
```

## Traceback
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/bercik/git/test/fbchat/fbchat/_client.py", line 485, in searchForThreads
    j = self.graphql_request(_graphql.from_query(_graphql.SEARCH_THREAD, params))
  File "/home/bercik/git/test/fbchat/fbchat/_client.py", line 183, in graphql_request
    return self.graphql_requests(query)[0]
  File "/home/bercik/git/test/fbchat/fbchat/_client.py", line 175, in graphql_requests
    return tuple(self._post("/api/graphqlbatch/", data, as_graphql=True))
  File "/home/bercik/git/test/fbchat/fbchat/_client.py", line 135, in _post
    return _graphql.response_to_json(content)
  File "/home/bercik/git/test/fbchat/fbchat/_graphql.py", line 54, in response_to_json
    _util.handle_graphql_errors(value)
  File "/home/bercik/git/test/fbchat/fbchat/_util.py", line 148, in handle_graphql_errors
    error.get("code"), error.get("message"), error.get("debug_info")
AttributeError: 'NoneType' object has no attribute 'get'
```

## Environment information
- python: `3.7.3`
- `fbchat`: I tested on fresh pull. `1.7.0`

## Research
Facebook replies normally. Reply is perfectly fine.
```
"error": null
[...]
{
   "successful_results": 1,
   "error_results": 0,
   "skipped_results": 0
}

```
For example these two work fine
```
715: queries.append(_graphql.from_doc_id("2147762685294928", params))
837: j = self.graphql_request(_graphql.from_doc_id("1349387578499440", params))
```
And this doesn't.
```
445: j = self.graphql_request(_graphql.from_query(_graphql.SEARCH_USER, params))
```

**Comment by krzesu0** [comment]
It may be caused by *huge* outage of FB services here Europe.

**Comment by madsmtm** [comment]
Whoops, nope, this is definitely my fault, I accidentally introduced this in bc551a63c2249477f11545f75260bca6b0198c67. Fixed in 31d13f8fae9c6e5db381aa1ac743f2d3022a764f. Thanks for the quick report!
